### PR TITLE
chore(ARCH-466): start sb via talend/scripts in components and containers

### PR DESCRIPTION
**Pull request by jsomsanith-tlnd**
**What is the problem this PR is trying to solve?**

storybook setup is replicate in every packages with some parts in common, but dependencies and so adds maintenance burden.

**What is the chosen solution to this problem?**

Move storybook setup to ui-scripts has been done to obtain a zero config setup that fits our needs.
Now we just use it.

**Please check if the PR fulfills these requirements**

- [ ] The PR have used `yarn changeset` to a request a release from the CI if wanted.
- [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: packages/components/.storybook/i18n.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import { namespaces as tuiNamespaces } from '@talend/locales-tui-components/namespaces';
+import { locales as tuiLocales } from '@talend/locales-tui-components/locales';
+
+export const namespaces = [tuiNamespaces];
+export const locales = [tuiLocales];
```
Comment (jmfrancois) [inline packages/components/.storybook/i18n.js:5]: this is a regression from current master which load locales from unpkg and go againts #3692

File: packages/components/src/ActionBar/ActionBar.stories.js [hunk 2]
```diff
@@ -1,5 +1,4 @@
 import React from 'react';
-import { storiesOf } from '@storybook/react';
 import { action } from '@storybook/addon-actions';
```
Comment (jmfrancois) [inline packages/components/src/ActionBar/ActionBar.stories.js:2]: this looks weid to me to import package not specified in devDependencies 🤔

File: packages/components/package.json [hunk 3]
```diff
@@ -70,15 +70,11 @@
     "uuid": "^3.4.0"
   },
   "devDependencies": {
-    "@storybook/addon-a11y": "^6.4.19",
-    "@storybook/addon-actions": "^6.4.19",
```
Comment (jmfrancois) [inline packages/components/package.json:74]: this one is required by stories, we should keep it

Comment (jsomsanith-tlnd) [reply packages/components/package.json:74]: I find it hard if we upgrade sb in the future, libs and apps have to know that starting this version of talend/scripts we need to sync our addon-action version to a compatible version.
Just an idea: what about exposing that from @talend/scripts-config-storybook-lib ? Not sure it's a good one

File: packages/components/talend-scripts.json [hunk 4]
```diff
@@ -1,3 +1,6 @@
 {
-  "preset": "@talend/scripts-preset-react-lib"
+  "preset": "@talend/scripts-preset-react-lib",
+  "dynamic-cdn-webpack-plugin": {
+    "resolver": "@talend/module-to-cdn"
+  }
```
Comment (jmfrancois) [inline packages/components/talend-scripts.json:5]: why do you need this ? It is supposed to be the default value.

File: packages/components/package.json [hunk 5]
```diff
@@ -56,7 +56,7 @@
     "react-bootstrap": "0.32.4",
     "react-debounce-input": "3.2.0",
     "react-draggable": "^4.4.4",
-    "react-grid-layout": "0.16.6",
+    "react-grid-layout": "^1.3.3",
```
Comment (jmfrancois) [inline packages/components/package.json:59]: Screenshot 2022-03-01 at 15 25 20

😂

Comment (jsomsanith-tlnd) [reply packages/components/package.json:59]: yeah.... 😅

File: .github/workflows/pr-demo.yml [hunk 6]
```diff
@@ -32,6 +32,7 @@ jobs:
       - name: Install and build
         run: |
           yarn install --frozen-lockfile
+          yarn pre-release
```
Comment (jmfrancois) [inline .github/workflows/pr-demo.yml:35]: 😬 comes from the common cdn config

**Review by jmfrancois — APPROVED** [review]
just question around API.
looks good to me to point to storybook actions.
just have to check duplicate at upgrade times but seems acceptable.
Big thanks for this hard work Jimmy !

**Comment by frassinier** [comment]
http://3665.talend.surge.sh/icons is not fully exhaustive

File: .github/workflows/pr-demo.yml (hunk 6 above)
Comment (frassinier) [reply .github/workflows/pr-demo.yml:35]: why does it matter here?

File: packages/components/package.json (hunk 3 above)
Comment (frassinier) [reply packages/components/package.json:74]: In fact, actions are already in essentials package

File: packages/components/src/ActionBar/ActionBar.stories.js (hunk 2 above)
Comment (frassinier) [reply packages/components/src/ActionBar/ActionBar.stories.js:2]: I can live with an exclusion from eslint at this point

**Comment by frassinier** [comment]
Amazing work! Well done!

**Review by frassinier — APPROVED** [review]

File: packages/components/.storybook/i18n.js [hunk 7]
```diff
@@ -0,0 +1,8 @@
+import { namespaces as tuiNamespaces } from '@talend/locales-tui-components/namespaces';
+import { namespaces as dsNamespaces } from '@talend/locales-design-system/namespaces';
+
+export const namespaces = [...tuiNamespaces, ...dsNamespaces];
+export const remoteLocalesMap = {
+	'tui-components': 'https://unpkg.com/@talend/locales-tui-components/locales/{{lng}}/{{ns}}.json',
+	'design-system': 'https://unpkg.com/@talend/locales-design-system/locales/{{lng}}/{{ns}}.json',
+};
```
[the remark this answers is not in the document]
Comment (jsomsanith-tlnd) [reply packages/components/.storybook/i18n.js:8]: It's configured in preview (i18n conf was imported at preview time) now

**Pull request merged**